.github/workflows/agent-lint-and-test.yaml

```diff
@@ -1,12 +1,10 @@
 name: Lint and Test Prefect Agent Chart
-
 on:
   pull_request:
     paths:
       - .github/workflows/agent-lint-and-test.yaml
       - .github/linters/agent-ct.yaml
       - 'charts/prefect-agent/**'
-
```

Comment:
@jamiezieziula updatecli seemed to remove these, and I have no idea how to control that behavior

Reply:
Merging for now, if we figure out some way to fix updatecli, we can bring these back